plugins/techdocs/src/reader/components/TechDocsHome.tsx

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import { Grid } from '@material-ui/core';
+import { ItemCard } from '@backstage/core';
+import { TechDocsPageWrapper } from './TechDocsPageWrapper';
+
+export const TechDocsHome = () => {
+  const navigate = useNavigate();
+
+  return (
+    <>
+      <TechDocsPageWrapper
+        title="Documentation"
+        subtitle="Documentation available in Backstage"
+      >
+        <Grid container data-testid="docs-explore">
+          <Grid item xs={12} sm={6} md={3}>
+            <ItemCard
```

Comment:
Could we even move this to an `const documentationSites = [...]` object and map through them here? It would easier for us to add sites, as well as automate it to read this from an API in the future?